tiup/tiup-command-mirror-clone.md

```diff
@@ -0,0 +1,48 @@
+---
+title: tiup mirror clone
+---
+
+# tiup mirror clone
+
+The command `tiup mirror clone` is used to clone an existing mirror or clone some of its components to create a new mirror. The new mirror has the same components as the old one, but uses different signature keys.
+
+## Syntax
+
+```sh
+tiup mirror clone <target-dir> [global version] [flags]
+```
+
+- `<target-dir>` is used to set the local path to the cloned mirror. If the path does not exist, TiUP automatically creates one.
+- If `[global version]` is specified, TiUP tries to clone all components of the specified version. If some components do not have the specified version, then clone its latest version.
+
+## Flags
```

Comment:
Options or Flags?

Reply:
Seems like both are OK, but "options" may be a better choice according to https://superuser.com/questions/1070059/is-there-a-difference-between-a-command-line-flag-and-a-command-line-option. @lucklove Could you help us confirm this?

Reply:
options is better, I think

Reply:
Updated in https://github.com/pingcap/docs/pull/4965/commits/daa400580ae8a23d2fafb924b7bd21e7d8ebdd17.